twake/frontend/src/app/features/users/hooks/use-online-users.ts

```diff
@@ -52,18 +56,26 @@ export const useOnlineUsers = (): void => {
 
   useEffect(() => {
     let id: any;
+    let setId: any;
     if (websocket) {
       // got some local users, ask if there are still online
-      const api = OnlineUserRealtimeAPI(websocket);
+      const api = OnlineUserRealtimeAPI(websocket, companyId);
       id = setInterval(async () => {
         const users = onlineUsers.map(u => u.id).filter(<T>(n?: T): n is T => Boolean(n));
         const status = await api.getUsersStatus(users);
         updateOnline(status);
       }, INTERVAL);
+
+      setId = setInterval(async () => {
```

Comment:
Let's rename id and setId to getIntervalId and setIntervalId.
Also move lines 58 and 59 to the top of the file, like line 19.